.aws/src/main.ts

```diff
@@ -87,6 +91,35 @@ class PocketEventBus extends TerraformStack {
       'forgot-password-request-event-schema'
     );
   }
+
+  /**
+   * Create PagerDuty service for alerts
+   * @private
+   */
+  private createPagerDuty() {
```

Comment:
in future, if we want to spend alerts across different squads/teams,

we can just create pagerDuty handler for that paritcular squad and pass them to different event-rule stacks from main.ts

Reply:
if this is a repeat thing we could consider a helper function in terraform